topics/geneontology/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: GO, geneontology, gene-ontology
+display_name: The Gene Ontology Consortium
+github_url: https://github.com/geneontology
+logo: GO-logo-vertical.png
```

Comment:
This image needs to be included in this directory.

Reply:
Is there a way to do that on the web or do I need to use some sort of GitHub client? Normally I'm able to upload files via the web interface but that is not allowed in this directory.

Reply:
Yes, you should be able to do it through the branch for this PR on your fork of github/explore. Here is the direct link to the directory you'll want to upload the file to: https://github.com/nlharris/explore/tree/patch-1/topics/geneontology

Once you are on that page, you can click on "Add file" then "Upload files":
![image](https://user-images.githubusercontent.com/4008677/128674513-c3925348-077c-4771-b782-9123238533a8.png)

You'll want to be sure that the PNG file's name is the same as the topic/folder, so for this, you'll want the file name to be `geneontology.png`.

Once the upload is complete, you can commit directly to your `patch-1` branch, and that will update this PR. Once the PR is updated, I can approve the test runs and we can get this merged in as long as all the other tests pass :-D

Reply:
Thanks so much for the detailed instructions (and particularly the URL)!

Reply:
👋🏻 Hi, this is still failing because of this bit:

> You'll want to be sure that the PNG file's name is the same as the topic/folder, so for this, you'll want the file name to be `geneontology.png`.

You'll need to [rename the image file](https://docs.github.com/en/github/managing-files-in-a-repository/managing-files-on-github/renaming-a-file) to `geneontology.png` and change the filename in `index.md` to match.

Reply:
Thanks! I couldn't see a way to change the name of the image file but I deleted the image file and added a new one with the desired name.